lib/kubernetes-deploy/container_logs.rb

```diff
@@ -72,9 +72,6 @@ def deduplicate(logs)
     def split_timestamped_line(log_line)
       timestamp, message = log_line.split(" ", 2)
       [Time.parse(timestamp), message]
-    rescue ArgumentError
```

Comment:
I'm tempted to put this back. I don't like the idea of this blowing up just because of a parsing error...